src/components/Projects/index.tsx

```diff
@@ -1,23 +1,38 @@
 'use client';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { ProjectsBanner } from './ProjectsBanner';
 import { useFetchAllProjects } from '@/hooks/useFetchAllProjects';
 import { ProjectHoverCard } from '../ProjectCard/ProjectHoverCard';
 import RoundCountBanner from '../RoundCountBanner';
 import { useFetchActiveRoundDetails } from '@/hooks/useFetchActiveRoundDetails';
+import { getMostRecentEndRound } from '@/helpers/round';
 
 const projectCardStyle = '';
 
 export const ProjectsView = () => {
   const { data: allProjects, isLoading } = useFetchAllProjects();
   const { data: activeRoundDetails } = useFetchActiveRoundDetails();
+  const [isRoundEnded, setIsRoundEnded] = useState(false);
+
+  useEffect(() => {
+    const fetchMostRecentEndRound = async () => {
```

Comment:
@lovelgeorge99 Do you think this name can show the logic better?
```suggestion
    const checkRoundEnded = async () => {
```